frontend/components/events/create/DescriptionImageForm.tsx

```diff
@@ -19,12 +20,38 @@ export function DescriptionImageForm({
   setImagePreviewUrl,
   updateField,
 }: DescriptionImageFormProps) {
+  const [previousImages, setPreviousImages] = useState<string[]>([]);
+
+  useEffect(() => {
+    // Fetch previous images from Firebase and set them in state
+    fetchPreviousImagesFromFirebase().then((images) => {
+      setPreviousImages(images);
+    });
+  }, []);
+
   const updateDescription = (e: string) => {
     updateField({
       description: e,
     });
   };
 
+  const fetchPreviousImagesFromFirebase = async () => {
+    // Fetch previous images from Firebase and return their URLs
+    // Example: You can use Firebase Firestore or Firebase Realtime Database to store image URLs
+    const images = await fetch('YOUR_FIREBASE_IMAGE_URLS_ENDPOINT');
```

Comment:
how does this work? is this meant to be a dud endpoint? we don't fetch from firestore with `http fetches`? we use the firestore sdk in the service layer. 

PS. this function could be moved to `imageService`